packages/web-components/src/compound-button/compound-button.styles.ts

```diff
@@ -0,0 +1,121 @@
+import { css } from '@microsoft/fast-element';
+import { styles as ButtonStyles } from '../button/button.styles.js';
+import {
+  colorNeutralForeground2,
+  colorNeutralForeground2BrandHover,
+  colorNeutralForeground2BrandPressed,
+  colorNeutralForeground2Hover,
+  colorNeutralForeground2Pressed,
+  colorNeutralForegroundDisabled,
+  colorNeutralForegroundOnBrand,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontWeightRegular,
+  lineHeightBase300,
+  lineHeightBase400,
+  spacingHorizontalS,
+  spacingHorizontalSNudge,
+  spacingHorizontalXS,
+} from '../theme/design-tokens.js';
+
+// Need to support icon hover styles
+export const styles = css`
+  ${ButtonStyles}
+
+  :host .control,
+  :host(:is([size])) .control {
+    gap: 12px;
+    height: auto;
+    padding-top: 14px;
+    padding-inline: 12px;
+    padding-bottom: 16px;
+    font-size: ${fontSizeBase300};
```

Comment:
The `fluent-link` PR made me go back to these button PRs same comment on font-size.

"We inherit on the host but set it here on in the .control class. This isn't ideal because you can't set font-size directly on the implementation of fluent-link and will need to use a ::part selector. Instead this should be inherit and the font-size: ${fontSizeBase300} should be set on the host."